interfaces/builtin/fwupd.go

```diff
@@ -111,6 +112,15 @@ const fwupdPermanentSlotAppArmor = `
       name="org.freedesktop.fwupd",
 `
 
+const fwupdPermanentSlotAppArmorClassic = `
+# Description: Allow operating as the fwupd service. This gives privileged
+# access to the Classic system.
+
+  # allow access to fwupd* and fw/ under any distro for classic systems
+  /boot/efi/EFI/*/fwupd*.efi* rw,
+  /boot/efi/EFI/*/fw/** rw,
```

Comment:
As a side note, what about accessing `/efi`? AFAICT some systems have started using /efi instead of /boot/efi. Can fwupd detect such scenario and handle it correctly?

Reply:
IIRC fwupd indeed detected common ESP such as /efi or/boot/efi, but maybe @superm1 can enumerate how many cases will be perfect here.

Reply:
It used to be a hardcoded list and heuristics to go with them (IIRC `/efi`, `/boot/efi` and `/boot`), but fwupd by default now uses udisks2 to determine location of the ESP and mount it using udisks2 if it's not already mounted.

I think the most common case is probably `/boot/efi`.